config/routes.rb

```diff
@@ -5,7 +5,7 @@
     skip_controllers :authorizations, :authorized_applications, :tokens, :token_info
   end
 
-  devise_for :users
+  devise_for :users, :controllers => { :registrations => 'registrations' }
```

Comment:
```suggestion
  devise_for :users, controllers: { registrations: :registrations }
```

Reply:
I got trapped by devise tutorial!!! I should not trust in copy/paste anymore with Ruby :P

Reply:
Fixed.